CDAP-13925 fix tag value of profile

yaojiefeng opened the pull request:
JIRA: https://issues.cask.co/browse/CDAP-13925
Build: https://builds.cask.co/browse/CDAP-RUT1602-1

Fix a tag value when we emit profile metrics, changed the PROFILE tag to use profile name instead of scoped name since we already have a PROFILE_SCOPE tag.

Note that this will require corresponding UI changes to make sure metrics can be correctly retrieved.

yaojiefeng committed:
CDAP-13925 fix tag value of profile

albertshau reviewed (commented):
lgtm. Let's make sure the corresponding UI changes are done on this branch too.

yaojiefeng commented:
@tonybach can you make the corresponding UI change in this branch?

ajainarayanan committed:
Fixes profile name for context while fetching metrics in UI

 - Fixes adding system namespace to metrics tags when querying for system profile metrics
 - Fixes profiles to metrics map to differentiate between system and user profiles with same name
 - Fixes profiles preview to show the right metrics

elfenheart commented:
👍 ui code LGTM

elfenheart commented:
👍 ui LGTM

elfenheart reviewed (approved):
👍 ui LGTM

yaojiefeng commented:
Tested with the latest UI pack, did not find any issue, we can merge this
